src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/TrackBars.cs

```diff
@@ -29,7 +29,17 @@ private void chbRightToLeft_CheckedChanged(object sender, EventArgs e)
             trackBar1.RightToLeft = chbRightToLeft.Checked ? RightToLeft.Yes : RightToLeft.No;
         }
 
-        private void numericMinimum_ValueChanged(object sender, EventArgs e)
+        private void tickstyleNone_CheckedChanged(object sender, EventArgs e)
+        {
+            if (trackBar1.TickStyle == TickStyle.BottomRight)
+            {
+                trackBar1.TickStyle = System.Windows.Forms.TickStyle.None;
+            }
+            else
+            { trackBar1.TickStyle = System.Windows.Forms.TickStyle.BottomRight; }
```

Comment:
We do not generally use fully qualified names except in auto-generated code
```suggestion
            { trackBar1.TickStyle = TickStyle.BottomRight; }
```